packages/teams-js/src/internal/validOrigins.ts

```diff
@@ -14,27 +15,40 @@ function isValidOriginsCacheEmpty(): boolean {
   return validOriginsCache.length === 0;
 }
 
-async function getValidOriginsListFromCDN(): Promise<string[]> {
-  if (!isValidOriginsCacheEmpty()) {
+async function getValidOriginsListFromCDN(disableCache?: boolean): Promise<string[]> {
```

Comment:
>disableCache?: boolean [](http://example.com/codeflow?start=42&length=22)

Consider:
1. Naming this `shouldDisableCache` makes the intent clearer
2. Set a default value for this, e.g., `shouldDisableCache? :boolean = false` so that the default behavior is clear to callers and the consumers in the function don't have to to be concerned with any `undefined weirdness.